packages/tools/locales/fr_FR.po

```diff
@@ -854,55 +813,48 @@ msgstr "Impossible d'initialiser la synchronisation."
-"Impossible de charger module \"%s\" pour le format d'entrée \"%s\" et de "
-"sortie \"%s\""
+"Impossible de charger module \"%s\" pour le format d'entrée \"%s\" et de sortie \"%s\""
 
 #: packages/lib/services/interop/InteropService.ts:276
 msgid "Cannot load \"%s\" module for format \"%s\" and target \"%s\""
 msgstr ""
-"Impossible de charger module \"%s\" pour le format d'entrée \"%s\" et de "
-"sortie \"%s\""
+"Impossible de charger module \"%s\" pour le format d'entrée \"%s\" et de sortie \"%s\""
 
 #: packages/lib/models/Note.ts:630
 msgid "Cannot move note to \"%s\" notebook"
 msgstr "Impossible de déplacer la note vers le carnet \"%s\""
 
-#: packages/app-mobile/components/screens/folder.js:73
-#: packages/lib/models/Folder.ts:883
+#: packages/app-mobile/components/screens/folder.js:73 packages/lib/models/Folder.ts:883
 msgid "Cannot move notebook to this location"
 msgstr "Impossible de déplacer le carnet à cet endroit"
 
 #: packages/lib/onedrive-api.ts:468
 msgid ""
-"Cannot refresh token: authentication data is missing. Starting the "
-"synchronisation again may fix the problem."
+"Cannot refresh token: authentication data is missing. Starting the synchronisation again "
+"may fix the problem."
 msgstr ""
-"Impossible de rafraîchir la connexion à OneDrive. Démarrez la "
-"synchronisation à nouveau pour corriger le problème."
+"Impossible de rafraîchir la connexion à OneDrive. Démarrez la synchronisation à nouveau "
+"pour corriger le problème."
 
 #: packages/server/src/models/UserModel.ts:246
 msgid "Cannot save %s \"%s\" because it is larger than the allowed limit (%s)"
-msgstr ""
-"Impossible d'enregistrer l'élément %s \"%s\" car il excède la limite allouée "
-"(%s)"
+msgstr "Impossible d'enregistrer l'élément %s \"%s\" car il excède la limite allouée (%s)"
 
 #: packages/server/src/models/UserModel.ts:263
 msgid ""
-"Cannot save %s \"%s\" because it would go over the total allowed size (%s) "
-"for this account"
+"Cannot save %s \"%s\" because it would go over the total allowed size (%s) for this account"
 msgstr ""
-"Impossible d'enregistrer l'élément %s \"%s\" car cela excèderait la limite "
-"totale allouée (%s) pour ce compte"
+"Impossible d'enregistrer l'élément %s \"%s\" car cela excèderait la limite totale allouée "
+"(%s) pour ce compte"
 
 #: packages/lib/services/share/ShareService.ts:338
 msgid ""
-"Cannot share encrypted notebook with recipient %s because they have not "
-"enabled end-to-end encryption. They may do so from the screen Configuration "
-"> Encryption."
+"Cannot share encrypted notebook with recipient %s because they have not enabled end-to-end "
+"encryption. They may do so from the screen Configuration > Encryption."
 msgstr ""
-"Impossible de partager ce carnet chiffré car le destinataire %s car ils "
-"n'ont pas activé le chiffrement. Ils peuvent le faire à partir de l'écran "
-"Configuration > Chiffrement."
+"Impossible de partager ce carnet chiffré car le destinataire %s n'a pas activé le "
```

Comment:
I must think this is for multiple recipients, hence probably say "car les destinataires %s n'ont pas activé"